Lib/xml/etree/ElementTree.py

```diff
@@ -196,12 +197,35 @@ def copy(self):
         original tree.
 
         """
+        warnings.warn(
+            "elem.copy() is deprecated. Use copy.copy(elem) instead.",
+            DeprecationWarning
+            )
+        return self.__copy__()
+
+    def __copy__(self):
         elem = self.makeelement(self.tag, self.attrib)
         elem.text = self.text
         elem.tail = self.tail
         elem[:] = self
         return elem
 
+    def __deepcopy__(self, memo):
```

Comment:
This method is not needed in Python implementation.

Reply:
I would prefer you have this discussion on the bpo ticket, as PEP 399 suggests otherwise.

Reply:
This doesn't contradict PEP 399.

Reply:
The intention of PEP-399 is that both implementations *behave* the same. It's not that they use exactly the same mechanisms to reach that goal. Some things work in Python that don't easily work in C, and vice-versa. That's why there *are* two implementations in the first place.

If we can avoid adding Python (or C) code without sacrificing that goal, then that is a good thing. Code that does not exist is entirely free of bugs, for example. :)